chromium_src/chrome/browser/permissions/chrome_permissions_client.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_PERMISSIONS_CHROME_PERMISSIONS_CLIENT_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_PERMISSIONS_CHROME_PERMISSIONS_CLIENT_H_
+
+#define BRAVE_CHROME_PERMISSIONS_CLIENT_H_                               \
```

Comment:
we don't need to add a new method here, `CanBypassEmbeddingOriginCheck` is already virtual so just subclass `ChromePermissionsClient`. I think in this case it would be best to rename the entire original class so all calls to `ChromePermissionsClient::GetInstance()` will work without modification

Reply:
Hm, the original method does not have the permission type passed in which we need, that's why I use another method here.

Reply:
I didn't notice that, but see https://github.com/brave/brave-core/pull/9443/files#r670740026

Reply:
Fixed in https://github.com/brave/brave-core/pull/9443/commits/b127eefe84bac3157ee00dbdfef764ebf6a691a6